pkg/metrics/statsd/golang/metrics.go

```diff
@@ -34,11 +35,11 @@ func NewMetrics(log *logrus.Entry, m metrics.Interface) (Runnable, error) {
 		r:   prometheus.NewRegistry(),
 	}
 
-	if err := g.r.Register(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{})); err != nil {
+	if err := g.r.Register(collectors.NewProcessCollector(collectors.ProcessCollectorOpts{})); err != nil {
```

Comment:
`github.com/prometheus/client_golang/prometheus` got updated and `prometheus.NewProcessCollector` while still exists - it is depricated. Our `golangci-lint` job complains about it.